newsView.js

```diff
@@ -0,0 +1,11 @@
+class NewsView {
+  addHeadline(headline) { 
+    const headlinesContainerEl = document.querySelector('#headline-container');
+    const headlineEl = document.createElement('div');
+    headlineEl.innerText = headline;
+    headlineEl.className = 'headline';
```

Comment:
Really smart to use a class 'headline' to be able to target any headlines down the line and test accordingly!